### ISSUE TITLE
Color contrast is reporting incorrect error

### ISSUE THREAD

**Issue by davidlower** [GoogleChrome/lighthouse#8445]
I have noticed that the lighthouse tool has been a bit temperamental recently in the color contrast ratio testing. Sometimes I test a website and the color contrast part of accessibility passes. Other times it doesn't. Same website, same color. 

However this recent one is clearly a bug. I have a particular website using blue as the main color (#0480d9) and it is marking it as failing on multiple areas where this color is being used. However ALL my other testing tools for color contrast are showing this color is passing with ease AA+ 4.12. I am not sure what is going on. I have darkened the blue so much and it is still failing.

I would appreciate your thoughts. Thank you for your time.
https://sigilower.com/

![Screenshot 2019-04-19 at 16 53 15](https://user-images.githubusercontent.com/29481254/56432216-73f41980-62c4-11e9-807a-e6a25cf42819.png)
![Screenshot 2019-04-19 at 17 04 09](https://user-images.githubusercontent.com/29481254/56432423-30e67600-62c5-11e9-9755-13da10df6295.png)

![Screenshot 2019-04-19 at 17 02 18](https://user-images.githubusercontent.com/29481254/56432353-e8c75380-62c4-11e9-9529-443c33ff076f.png)

**Comment by patrickhulce** [comment]
Thanks for filing @davidlower! 

From the [docs](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=AxeChrome)...

> Ensure color contrast of at least 4.5:1 for small text or 3:1 for large text, even if text is part of an image. Large text has been defined in the requirements as 18pt or 14pt bold.

In this case the text is not large enough to qualify for 3:1 and must instead be at least 4.5:1 which it is not, so it fails.

We use what [axe](https://github.com/dequelabs/axe-core/) says, so if you have an issue with their logic, I'd suggest filing it in their repo :)
